pyproject.toml

```
@@ -21,7 +21,7 @@ python-dateutil = "^2.8"
 
 [tool.poetry.dev-dependencies]
 asv = "^0.4.2"
-black = "^20.8b1"
+black = "^21.11b0"
```

This version of black is bugged iirc

---

```suggestion
black = "^21.11b1"
```
The old version got yanked